src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -116,13 +110,8 @@ export const SchemaDrivenAnnotationEditor = (
     entityId!,
     {
       enabled: !!entityId,
-      retry: false,
       refetchOnWindowFocus: false,
-      onError: e => {
-        if (e.status !== 404) {
-          handleError(e)
-        }
```

Comment:
404 here should not error, added `allowNotFoundError` to the corresponding SynapseClient method.